tests/unit/__init__.py

```diff
@@ -162,7 +162,7 @@ def setUp(self):
         # Disable Sentry reporting for tests, otherwise they'll hang waiting
         # for input
         self.useFixture(
-            fixtures.EnvironmentVariable("SNAPCRAFT_ENABLE_SENTRY", "false")
+            fixtures.EnvironmentVariable("SNAPCRAFT_ENABLE_ERROR_REPORTING", "false")
```

Comment:
oh, the new structure makes this unnecessary, to be honest, I'd remove these lines completely